src/AndroidDebugLauncher/NdkReleaseId.cs

```diff
@@ -189,7 +272,7 @@ public override string ToString()
                 }
 
                 object subRelease = string.Empty;
-                if (this.SubRelease != (char)0)
+                if (this.SubRelease != (char)0 && this.SubRelease != 'a')
```

Comment:
NDK versioning never has a 'a'. The character for the subversion is only included if it's not 'a'.